funnel/transports/whatsapp.py

```diff
@@ -1,3 +1,111 @@
-"""Support functions for sending a WhatsApp message. Forthcoming."""
+"""Support functions for sending an Whatsapp messages."""
 
 from __future__ import annotations
+
+from typing import Union
+
+from models import PhoneNumber, PhoneNumberBlockedError
+import phonenumbers
+import requests
+
+from baseframe import _
+
+from .. import app
+from .exc import (
+    TransportConnectionError,
+    TransportRecipientError,
+    TransportTransactionError,
+)
+
+__all__ = ['send_wa_via_meta', 'send_wa_via_on_premise']
```

Comment:
Don't export any symbols from this module. They should always be used as:

```python
from funnel.transports import whatsapp  # Using relative import: from ..transports import whatsapp

whatsapp.send(…)
```

Reply:
A wrapper function called `send` must be added to this file. It will call one of the available methods, depending on which one is enabled. Priority: hosted, Exotel, Twilio, any other cloud provider, Meta cloud